src/eko/scale_variations/__init__.py

```diff
@@ -6,40 +6,41 @@
 
 import enum
 
+from ..io.types import ScaleVariationsMethod
 from . import expanded, exponentiated
 
 
 class Modes(enum.IntEnum):
-    """Enumerate scale Variation modes."""
+    """Enumerate scale variation modes."""
 
     unvaried = enum.auto()
     exponentiated = enum.auto()
     expanded = enum.auto()
 
 
-def sv_mode(s):
+def sv_mode(s: ScaleVariationsMethod) -> Modes:
     """Return the scale variation mode.
 
     Parameters
     ----------
-    s : str
+    s :
         string representation
 
     Returns
     -------
-    enum.IntEnum
-        enum representation
+    i :
+        int representation
 
     """
     if s is not None:
         return Modes[s.value]
     return Modes.unvaried
 
 
-class ModeMixin:
+class ScaleVariationModeMixin:
```

Comment:
as you can see I did it as a first thing, because I expected a name clash between "SV mode" and "ev mode" - that didn't happen in the end, but I prefer the explicit name anyway - so I suggest to keep the change unless there are reasons against

Reply:
well it's already scoped in the proper module, but fine to keep it